app/src/componentLibrary/tooltip/README.md

```diff
@@ -0,0 +1,47 @@
+## **Tooltip with optional title and content**
+
+### Tooltip props:
+
+- **children**: _node_, optional, default = null
+- **title**: _string_, optional, default = ''
+- **side**: _string_, optional, default = 'top'
+- **arrowPosition**: _string_, optional, default = 'left'
```

Comment:
`Side: top` little bit confusing. Maybe be better to change to `placement` or something similar. No need for any action now, let's see the opinion of other team members